charts/neon-proxy/README.md

```diff
@@ -60,8 +60,9 @@ Kubernetes: `^1.18.x-x`
 | settings.authBackend | string | `"link"` | auth method used (console|link|postgres) |
 | settings.authEndpoint | string | `""` | auth endpoint, e.g. "http://console.neon/authenticate_proxy_request/" |
 | settings.domain | string | `""` | domain used in TLS cert for client postgres connections |
+| settings.sentryURL | string | `""` | url used by sentry to collect error/panic events in neon-proxy |
 | settings.uri | string | `""` |  |
 | tolerations | list | `[]` | Tolerations for pod assignment. |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.9.1](https://github.com/norwoodj/helm-docs/releases/v1.9.1)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```

Comment:
```suggestion
Autogenerated from chart metadata using [helm-docs v1.9.1](https://github.com/norwoodj/helm-docs/releases/v1.9.1)
```

Reply:
This is pretty annoying feature of helm-docs. We should probably upgrade to the latest version, but in the meantime this is probably the way to go.

Reply:
yeah I was getting stuck in a catch-22 situation with helm-docs

> Documentation not up to date. Please run helm-docs and commit changes!

and after running helm-docs, it changed the version in the neon-storage-broker as well.